frontend/src/hooks/api/getters/useAdminCount/useAdminCount.ts

```diff
@@ -0,0 +1,29 @@
+import useSWR from 'swr';
+import { formatApiPath } from 'utils/formatPath';
+import handleErrorResponses from '../httpErrorResponseHandler';
+
+export interface IAdminCount {
+    password: number;
+    no_password: number;
```

Comment:
Maybe nitpicking but should we use camelCase here? I know this is coming from the database table which means our type is coupled with the DB structure and that's fine at the store level but it shouldn't leak into the API